01-JS-DOM-APIs/index.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Index</title>
+    <link rel="stylesheet" href="css/style.css">
+    <script src="js/index.js"></script>
+  </head>
+  <body onload="fade()">
+    <section class="content">
+      <div class="search">
+        <label for="keyword-repositories">Search repositories: </label>
+        <input type="text" name="keyword-repositories" id="keyword-repositories">
+        <button type="button" id="button-get-repositories">Get repositories</button>
+        <span id="search-status"></span>
+      </div>
+      <div class="results">
+        <section class="hidden" id="fade-in">

```

Comment:
because I add style to the class with css and I need to identify the element with JS